utils/schema/artifact.json

```
@@ -0,0 +1,194 @@
+    },
+    "datasources": {
+      "type": "array",
+      "items": { "$ref": "#/$definitions/datasource" }
+    },
+    "alerts": {
+      "type": "array",
+      "items": { "$ref": "#/$definitions/alert" }
+    },
+    "dashboards": {
+      "type": "array",
+      "items": { "$ref": "#/$definitions/dashboard" }
+    },
+    "datasourceIds": { "enum": [] }
+  },
+  "required": [
+    "quickstarts",
+    "datasources",
+    "alerts",
+    "dashboards",
+    "datasourceIds"
+  ],
+  "additionalProperties": false,
+
+  "$definitions": {
+    "quickstart": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "description": { "type": "string" },
+        "summary": { "type": "string" },
+        "title": { "type": "string" },
+        "authors": {
+          "type": "array", "items": { "type": "string" }
+        },
+        "documentation": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": { "type": "string" },
+              "description": { "type": "string" },
+              "url": { "type": "string" }
+            },
+            "required": ["name", "description", "url"],
+            "additionalProperties": false
+          }
+        },
+        "level": { "enum": ["New Relic", "Community", "Verified"] },
+        "icon": { "type": "string" },
+        "keywords": {
+          "type": "array", "items": { "type": "string" }
+        },
+        "slug": { "type": "string" },
+        "alertPolicies": {
+          "type": "array", "items": { "type": "string" }
+        },
+        "dashboards": {
+          "type": "array", "items": { "type": "string" }
+        },
+        "datasourceIds": {
+          "type": "array", "items": { "type": "string" }
+        }
+      },
+      "required": [
+        "id",
+        "description",
+        "summary",
+        "title",
+        "authors",
+        "documentation",
+        "level",
+        "icon"
+      ],
+      "additionalProperties": false
+    },
+
+    "datasource": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "displayName": { "type": "string" },
+        "description": { "type": "string" },
+        "install": {
+          "type": "object",
+          "properties": {
+            "primary": { "$ref": "#/$definitions/installDirective" },
+            "fallback": { "$ref": "#/$definitions/installDirective" }
+            }
+          },
+          "required": ["primary"],
+          "additionalProperties": false
+        },
+        "keywords": {
+          "type": "array", "items": { "type": "string" }
+        },
+        "categoryTerms": {
+          "type": "array", "items": { "type": "string" }
+        },
+        "icon": { "type": "string" }
+      },
+      "required": ["id", "displayName", "install"],
+      "additionalProperties": false
+    },
+
+    "installDirective": {
+      "enum", [
```

```suggestion
      "enum": [
```
Should this be changed?

---

Yes?